config-service-impl/src/main/java/org/hypertrace/config/service/store/DocumentConfigStore.java

```diff
@@ -55,7 +55,7 @@
   @Override
   public void init(Config config) {
     datastore = initDataStore(config);
-    this.collection = getOrCreateCollection(datastore);
+    this.collection = this.datastore.getCollection(CONFIGURATIONS_COLLECTION);;
```

Comment:
Won't this affect other stores ?

Reply:
Added the context and description. 
Postgres handles currently in getCollection API
- https://github.com/hypertrace/document-store/blob/main/document-store/src/main/java/org/hypertrace/core/documentstore/postgres/PostgresDatastore.java#L116

Reply:
Have tested locally running Hypetrace with postgres
<img width="1657" alt="Screenshot 2021-02-01 at 5 38 57 PM" src="https://user-images.githubusercontent.com/53209990/106457095-7c770d80-64b4-11eb-819f-5d953d5eb2f2.png">
<img width="1672" alt="Screenshot 2021-02-01 at 5 40 17 PM" src="https://user-images.githubusercontent.com/53209990/106457166-944e9180-64b4-11eb-819c-8d395ffd14b2.png">

Reply:
We had a restructuring ticket for the same, but it was not prioritized so far - https://github.com/hypertrace/hypertrace/issues/123